ovos_audio/service.py

```
@@ -475,15 +482,40 @@ def handle_stop(self, message: Message):
+        candidates = [normalized_uri]
+        alias = _DEFAULT_SOUND_ALIASES.get(normalized_uri)
+        if alias and alias not in candidates:
+            candidates.append(alias)
+
+        for candidate in candidates:
+            resource_parts = ("res", *candidate.split("/"))
+            for package in _DEFAULT_SOUND_PACKAGES:
+                try:
+                    resource = files(package).joinpath(*resource_parts)
+                except Exception:
+                    continue
```

Ruff (0.15.5)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)